src/Umbraco.Web.BackOffice/Controllers/DictionaryController.cs

```diff
@@ -191,6 +193,40 @@ public ActionResult<DictionaryDisplay> GetById(Udi id)
             return _umbracoMapper.Map<IDictionaryItem, DictionaryDisplay>(dictionary);
         }
 
+        /// <summary>
+        /// Change the sort order for media
+        /// </summary>
+        /// <param name="move"></param>
+        /// <returns></returns>
+        public IActionResult PostMove(MoveOrCopy move)
+        {
+            var dictionaryItem =
+                _localizationService.GetDictionaryItemById(int.Parse(move.Id.ToString(), CultureInfo.InvariantCulture));
+
+            var parent =
+                _localizationService.GetDictionaryItemById(int.Parse(move.ParentId.ToString(), CultureInfo.InvariantCulture));
+
```

Comment:
@Lantzify you don't need to parse to and `int` here. `ParentId` and `Id` in `MoveOrCopy` are already defined as `int`.
https://github.com/umbraco/Umbraco-CMS/blob/v9/contrib/src/Umbraco.Core/Models/ContentEditing/MoveOrCopy.cs#L12-L24

Reply:
@bjarnef I belive I looked how the `PostSave` was done. And mixed up the type of `Id`. Sorry my bad!